packages/wrangler/src/index.tsx

```diff
@@ -1165,6 +1165,97 @@ function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
+		"secret:bulk [json]",
```

Comment:
This should be `"secret:bulk <json>"` as json is a required positional argument, rather than an optional one

Reply:
I can change it to angle brackets